scripts/coredump/gdbstubs/arch/arm_cortex_m.py

```diff
@@ -139,10 +139,10 @@ def handle_thread_register_group_read_packet(self):
             if stack_ptr_bytes is not None:
                 # Read registers stored at top of stack
                 stack_ptr = int.from_bytes(stack_ptr_bytes, "little")
-                barray = self.get_memory(stack_ptr, (size_t_size * 9))
+                barray = self.get_memory(stack_ptr, (size_t_size * 8))
 
                 if barray is not None:
-                    tu = struct.unpack(self.ARCH_DATA_BLK_STRUCT, barray)
+                    tu = struct.unpack("<IIIIIIII", barray)
```

Comment:
Can you squash these changes into previous commit where this functions is introduced?